static/app/views/alerts/rules/metric/utils/anomalyChart.tsx

```diff
@@ -0,0 +1,154 @@
+import type {MarkAreaComponentOption} from 'echarts';
+import moment from 'moment-timezone';
```

Comment:
idk how kosher this is, but it might be nice to use built in Intl / Datetime stuff rather than moment 😅

Reply:
Yeah dropping `moment` would be great! Going to punt on that until frontend tsc makes some decisions on https://github.com/getsentry/frontend-tsc/issues/69